client/state/data-getters/request-site-alerts.js

```diff
@@ -1,64 +0,0 @@
-import { requestHttpData } from 'calypso/state/data-layer/http-data';
-import { http } from 'calypso/state/data-layer/wpcom-http/actions';
-
-const requestSiteAlerts = ( siteId ) => {
-	const id = `site-alerts-${ siteId }`;
-
-	return requestHttpData(
-		id,
-		http(
-			{
-				method: 'GET',
-				path: `/sites/${ siteId }/alerts`,
-				apiNamespace: 'wpcom/v2',
-			},
-			{}
-		),
-		{
-			freshness: 5 * 60 * 1000,
-			fromApi: () => ( { suggestions, threats, warnings, updates } ) => [
-				[
-					id,
-					{
-						suggestions,
-						threats: threats.map( ( threat ) => ( {
-							id: threat.id,
-							signature: threat.signature,
-							description: threat.description,
-							firstDetected: Date.parse( threat.first_detected ),
```

Comment:
Seems like there is some response data massaging that we're removing here. While this particular field doesn't seem to be used, I just want to be sure we've ensured that none of those removed ones is in use.

Reply:
Yeah we only ever use `updates` fields, everything else is ignored.